vllm/distributed/device_communicators/shm_broadcast.py

```diff
@@ -1,12 +1,14 @@
 import os
-import pickle
+import struct
 import time
+import sys
 from contextlib import contextmanager
 from dataclasses import dataclass, field
 from multiprocessing import shared_memory
 from typing import List, Optional
 from unittest.mock import patch
 
+import msgspec
```

Comment:
curious if you could say more about what motivated this serialization format change?

Reply:
It's much faster